code/modules/projectiles/guns/projectile/revolver/wayfarer.dm

```diff
@@ -0,0 +1,24 @@
+//Sprite by Rebel
+/obj/item/gun/projectile/revolver/wayfarer
+	name = "\"Wayfarer\" caseless revolver"
+	desc = "A exstreamly well hand made revolver of a mix of steel, iron and plasteel with a polished wooden stock. \
```

Comment:
Here

Reply:
"A sleek; well-made revolver, hand-produced with reinforcements of plasteel, while the barrel assembly was a mix of high-carbon steel; low-carbon iron around the cylinder - and a polished mahogany handle."